lib/rules/one-var.js

```diff
@@ -250,6 +254,88 @@ module.exports = {
         }
 
 
+        /**
+         * Does the node have a `;` after it?
+         * @param   {ASTNode}  node The node to check
+         * @returns {boolean}       If the node has a `;` after it
+         */
+        function hasSemi(node) {
+            return sourceCode.getLastToken(node).type === "semi";
+        }
+
+        /**
+         * Fixer to split a VariableDeclaration into individual declarations
+         * @param   {VariableDeclaration}   declaration The `VariableDeclaration` to split
+         * @param   {?Function}             filter      Function to filter the declarations
+         * @returns {Function}                          The fixer function
+         */
+        function splitDeclarations(declaration, filter) {
+            if (!filter) {
+                filter = () => true;
+            }
+            return function *fix(fixer) {
```

Comment:
In https://github.com/eslint/eslint/pull/8101#issuecomment-287474842 it seemed like a few people didn't want to use generator functions in the codebase, although I'm personally fine with it.

Another option would be to make `fix` return an array, and use `.map` rather than a for-of loop.

Reply:
👍 I’ll use arrow functions!